init.rb

```diff
@@ -0,0 +1,18 @@
+require_relative "transport"
+require_relative "bike"
+require_relative "car"
+require_relative "delivery_service"
```

Comment:
Propose to make one file like `bootstrap.rb` and move all dependencies there. Because right now we need to require this file and it is going to puts some data and so on

Reply:
Created one file with dependencies